server/src/main/java/io/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -96,7 +98,8 @@ public RealtimeTuningConfig(
       @JsonProperty("indexSpec") IndexSpec indexSpec,
       @JsonProperty("persistInHeap") Boolean persistInHeap,
       @JsonProperty("ingestOffheap") Boolean ingestOffheap,
-      @JsonProperty("buffersize") Integer bufferSize
+      @JsonProperty("buffersize") Integer bufferSize,
+      @JsonProperty("aggregationBufferRatio") Float aggregationBufferRatio
```

Comment:
optional, but we could still write a dedicated RealtimeTuningConfigTest
LGTM otherwise